src/actions/authAction.js

```diff
@@ -62,10 +62,17 @@ export const validateResetToken = path => ({
 });
 
 
+const signup = formData => ({
+  type: types.SIGN_UP,
+  payload: axiosInstance()
```

Comment:
you can easily do `axiosInstance.post`, not sure why you're invoking axiosInstance

Reply:
The axioInstance is a function, not an object. 
` payload: axiosInstance().post('/users/', { user: formData }),`